packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -619,11 +620,6 @@ class _TextSelectionHandleOverlay extends StatefulWidget {
   }
 }
 
-/// The minimum size that a widget should be in order to be easily interacted
-/// with by the user.
-@visibleForTesting
-const double kMinInteractiveSize = 48.0;
```

Comment:
Unfortunately this change makes the PR backwards incompatible. It's a pretty minor one as these things go; but tests may break. We should follow the breaking change protocol for this PR.

Reply:
True.  I'll send out that email in a few minutes.